src/Modera/BackendToolsSettingsBundle/Contributions/ToolsSectionsProvider.php

```diff
@@ -16,17 +16,22 @@ class ToolsSectionsProvider implements ContributorInterface
 {
     private $items;
 
-    public function __construct()
+    /**
+     * ToolsSectionsProvider constructor.
```

Comment:
Please remove "ToolsSectionsProvider constructor.", and add to the class docblock:

#internal Since 2.56.0

Again, replace # with @.